src/main/java/frc/robot/controls/DriveController.java

```diff
@@ -11,12 +11,15 @@
  * Add your docs here.
  */
 public interface DriveController {
-    public double getMoveRequest();
+    Object getArmsSpeed = null;

```

Comment:
What is this? This is impossible, you can't have an instance variable on an interface